index.js

```diff
@@ -100,11 +100,20 @@ const displayPost = (post) => {
     <td class="post-posters">
       <div class="postersAvatars">${postersAvatars}</div>
     </td> 
-    <td class="post-replies">${post.posts_count-1}</td>
+    <td class="post-replies">${post.posts_count - 1}</td>
     <td class="post-views"></td>
     <td class="post-activity">${formatDateDiff(Date.now(), post.bumped_at)}</td>
   </tr>`;
   postsContainer.innerHTML += postRow;
 };
 
 copyright.innerText = new Date().getFullYear();
+
+let sortBtns = document.getElementsByName("sort");
```

Comment:
I would be nice if you could move this onto line 13 with another declarations and use `const` as its value should never change